demo_data/organizations.json

```diff
@@ -43,6 +43,28 @@
       "name": "north-pole",
       "state": "active",
       "title": "North Pole"
+    },
+    {
+      "users": [
+        {
+          "name": "south_pole_member",
+          "capacity": "member"
+        },
+        {
+          "name": "south_pole_editor",
+          "capacity": "editor"
+        },
+        {
+          "name": "south_pole_admin",
+          "capacity": "admin"
+        }
+      ],
+      "display_name": "South Pole",
+      "description": "South Pole demo organization",
+      "image_url": "https://media.gettyimages.com/vectors/south-pole-sign-with-santa-clause-vector-id165913270",
```

Comment:
This is an image url, and you have copied, pasted, found and replaced leading to the url being broken I think.  Might be worth setting this back to north-pole-sign-with-santa-clause-vector-id165913270 or even better finding a different image url for the south pole organisation.

Reply:
lol that's what I thought too when testing it, but check it out: https://media.gettyimages.com/vectors/south-pole-sign-with-santa-clause-vector-id165913270

it does a redirect using the id at the end presumably